README.md

```diff
@@ -83,7 +133,7 @@ MCT supports different quantization methods:
 | Quantization Method                           | Complexity | Computational Cost          |
 |-----------------------------------------------|------------|-----------------------------|
 | PTQ                                           | Low        | Low (order of minutes)      |
-| GPTQ (parameters fine-tuning using gradients) | Mild       | Mild (order of 2-3 hours)   |
+| GPTQ (parameters fine-tuning using gradients) | Moderate   | Moderate (order of 2-3 hours)   |
 | QAT                                           | High       | High (order of 12-36 hours) |
```

Comment:
the "Main features:" part below is unreadable, let's think how we can improve it

Reply:
Agree - will fix in next round.